test/withdraw.test.ts

```diff
@@ -362,6 +362,74 @@ describe('StakingPool.withdraw', () => {
         expect(poolDataAfterWithdraw).eql(expectedPoolDataWithdraw);
         expect(userDataAfterWithdraw).eql(expectedUserDataWithdraw);
       });
+
+
+      context('amdin set bob as manager', async () => {
+        it('bob becomes manager and call extend pool and alice withdraw', async () => {
```

Comment:
Why is 'alice withdraws' part necessary? I think the test in `settings.test.ts` is enough: when alice is set up as a manger by admin, she can extend the pool.